### PR TITLE
[Doc Update][Copilot in Teams branding]

### PR DESCRIPTION
**Pull request by v-ypalikila**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b564012b553856dbe9bb7d9b753d60bf0f944a39/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 87 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/57f77b57-fa4b-4261-82c1-fe552fec2902) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b564012b553856dbe9bb7d9b753d60bf0f944a39/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 90 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3faf263e-b2c6-4c41-9b37-61b26efcbfa5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-messagingextension-searchcommand-plugin.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b564012b553856dbe9bb7d9b753d60bf0f944a39/msteams-platform%2Fsbs-messagingextension-searchcommand-plugin.yml) | 84 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c88ff418-b70d-4b5f-bec9-d560afbafefc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-api-based-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/78e1861deaf7af689192328b7dd6c4d3cd632b49/msteams-platform%2Fmessaging-extensions%2Fbuild-api-based-message-extension.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/360ad415-dbc5-41c5-8743-2f23d49b1022) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/78e1861deaf7af689192328b7dd6c4d3cd632b49/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 87 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/498192f5-e877-4636-b3fa-940bd1bd9b5e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/78e1861deaf7af689192328b7dd6c4d3cd632b49/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 90 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87342bec-99da-4165-8398-d47b8c5fa844) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-messagingextension-searchcommand-plugin.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/78e1861deaf7af689192328b7dd6c4d3cd632b49/msteams-platform%2Fsbs-messagingextension-searchcommand-plugin.yml) | 84 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cebbbafd-0792-4f1b-93d7-fe540c6b62eb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Microsoft 365 plugins provide integration with various  Microsoft 365 products,
 
 We recommend that you build or upgrade your existing message extensions to maximize their usefulness and usability in Copilot for Microsoft 365. Message extensions should support one or more search commands, as these are recognized by Copilot for Microsoft 365 as skills it can execute on behalf of the user. Additionally, your extensions must meet the standards for compliance, performance, security, and user experience outlined in this article.
 
-:::image type="content" source="../assets/images/Copilot/ailib-copilot-interface.png" alt-text="Graphic shows the user experience between Microsoft Teams,Microsoft Copilot for Microsoft 365 and Microsoft 365 Chat.":::
+:::image type="content" source="../assets/images/Copilot/ailib-copilot-interface.png" alt-text="Graphic shows the user experience between Microsoft Teams, Microsoft Copilot for Microsoft 365 and M365 Chat.":::
```
Comment (erikadoyle) [inline msteams-platform/messaging-extensions/high-quality-message-extension.md:23]: This makes it sound like there are 3 experiences. For clarity, how about something like:

Graphic shows the user experience between Microsoft Teams and Copilot for Microsoft 365 (M365 Chat).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ The requirements for building message extension plugins for Copilot for Microsof
 
 ## Define descriptions
 
-A good description offers a clear and concise summary of the app’s features and allows Copilot for Microsoft 365 to efficiently discover and execute search operations. When a user enters the app name along with a verb, for example, **Find Contoso tickets**, the message extension plugin must be invoked from Microsoft 365 Chat.
+A good description offers a clear and concise summary of the app’s features and allows Copilot for Microsoft 365 to efficiently discover and execute search operations. When a user enters the app name along with a verb, for example, **Find Contoso tickets**, the message extension plugin must be invoked from Microsoft 365 Chat (M365 Chat).
```
Comment (erikadoyle) [inline msteams-platform/messaging-extensions/high-quality-message-extension.md:37]: How about:

must be invoked from Copilot for Microsoft 365 (M365 Chat)

File: msteams-platform/messaging-extensions/high-quality-message-extension.md [hunk 3]
```diff
@@ -503,7 +503,7 @@ Message extensions respond to a user input with an Adaptive Card. An Adaptive Ca
   * `Action.Execute`: Collects the input fields and sends them as a request to your bot service.
   * `Action.Submit`: Opens a Task module or Stage view using type invoke in data object.
 
-  :::image type="content" source="../assets/images/Copilot/ailib-copilot-action-buttons.png" alt-text="Graphic shows an example of the Update Stock, restock, and Cancel restock action buttons in an Adaptive Card response in the Microsoft 365 chat.":::
+  :::image type="content" source="../assets/images/Copilot/ailib-copilot-action-buttons.png" alt-text="Graphic shows an example of the Update Stock, restock, and Cancel restock action buttons in an Adaptive Card response in the M365 chat.":::
```
Comment (erikadoyle) [inline msteams-platform/messaging-extensions/high-quality-message-extension.md:506]: Adaptive Card response in M365 Chat (remove "the" and capitalize "Chat")

File: msteams-platform/sbs-messagingextension-searchcommand-plugin.yml [hunk 4]
```diff
@@ -49,7 +49,7 @@ items:
    | &nbsp; | [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app) | Microsoft Teams to collaborate with everyone you work with through apps for chat, meetings, and call-all in one place.|
    | &nbsp; | [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
    | &nbsp; | [Microsoft 365 developer account](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) | Access to Teams account with the appropriate permissions to install an app. |
-   | &nbsp; | Microsoft 365 Chat | The Microsoft 365 Chat feature in Copilot combines the power of large language models (LLMs) with the intelligence of the Microsoft Graph to help you get things done. |
+   | &nbsp; | Microsoft Copilot for Microsoft 365 | Plugins for Microsoft Copilot for Microsoft 365 are in preview and only work in Microsoft 365 Chat in Microsoft Teams. The Microsoft 365 Chat feature in Copilot combines the power of large language models (LLMs) with the intelligence of the Microsoft Graph to help you get things done.|
```
Comment (erikadoyle) [inline msteams-platform/sbs-messagingextension-searchcommand-plugin.yml:52]: Microsoft 365 Chat -> M365 Chat

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Looks good in general; left a couple suggestions

File: msteams-platform/sbs-messagingextension-searchcommand-plugin.yml (hunk 4 above)
Comment (v-ypalikila) [reply msteams-platform/sbs-messagingextension-searchcommand-plugin.yml:52]: Updated.

File: msteams-platform/messaging-extensions/high-quality-message-extension.md (hunk 3 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/high-quality-message-extension.md:506]: Updated.

File: msteams-platform/messaging-extensions/high-quality-message-extension.md (hunk 2 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/high-quality-message-extension.md:37]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d8b2f71a4ce926d831a8013c1d5e7877e62224d2/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 87 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6560fdfd-6fd9-498d-946e-f8551b1d3bc8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d8b2f71a4ce926d831a8013c1d5e7877e62224d2/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 90 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2eea81e0-bcca-4e23-a88b-633d5a2cb518) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-messagingextension-searchcommand-plugin.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d8b2f71a4ce926d831a8013c1d5e7877e62224d2/msteams-platform%2Fsbs-messagingextension-searchcommand-plugin.yml) | 84 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f8c2af22-a2a2-4596-9a62-8fdb3709dafd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/messaging-extensions/high-quality-message-extension.md]: Updated.

**Review by girliemac — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9b0ef0f55f445ed5e45af8b620ac12008b7ad37/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 85 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a15e4a8-ad07-4d41-9d9f-046429123dd8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9b0ef0f55f445ed5e45af8b620ac12008b7ad37/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 90 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83954896-8948-43d9-a0ac-ce35ac5bdb60) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-messagingextension-searchcommand-plugin.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b9b0ef0f55f445ed5e45af8b620ac12008b7ad37/msteams-platform%2Fsbs-messagingextension-searchcommand-plugin.yml) | 83 | 83 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/257ef09c-b9e7-4fad-b5dc-76284f8c1306) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**